portfolio/src/main/webapp/script.js

```diff
@@ -54,11 +54,28 @@ function handleResponse(response) {
 
 function addTextToDom(text) {
     console.log("Adding text to DOM: "+text);
-    const td = document.getElementById("text");
+    const table = document.getElementById("comment-section");
     const parsedText = JSON.parse(text); // text is a string and not an array
-    td.innerText = "";
-    var i;
+
+    var i; 
     for (i = 0; i < parsedText.length; i++) {
-        td.appendChild(document.createTextNode(parsedText[i] + "\n"));
+        const comment = parsedText[i];
+        const heading = ` (posted at ${comment.date})`;
+        const row = makeComment(comment.name + heading, comment.content);
+        table.appendChild(row);
     }
+}
+
+function makeComment(name, comment) {
```

Comment:
nit: I might rename these inputs to something like header and content so that it is clear they are not exactly comment.name and the comment object.

Reply:
done!